composer.json

```diff
@@ -9,7 +9,7 @@
         "php": ">=5.6"
     },
     "require-dev": {
-        "phpunit/phpunit": "3.7.*"
+        "phpunit/phpunit": "5.7.*"
```

Comment:
How about setting the `^5.7 || ^6.5` to support the `php7.2`?

Please refer the [official PHPUnit website](https://phpunit.de/).

Reply:
Will do!